tests/Unit/Domain/Creators/Test_Cylinder.cpp

```diff
@@ -55,79 +55,83 @@ void test_cylinder_construction(
        Direction<3>::upper_zeta()}});
```

Comment:
Please squash this commit into the one that changes the Cylinder, so the tests pass at every point in the commit history.